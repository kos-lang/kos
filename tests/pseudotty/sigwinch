# Run some command
1 + 2 + 3\r

# Put a long string in the command line
12345 + 23456 + 34567 + 45678

# Resize the window to 10 columns
#resize 10

\x7F # Backspace
\r

# Run the first command
\x10\x10\r

# Exit
\x04
