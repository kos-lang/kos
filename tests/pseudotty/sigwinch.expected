R:Kos 0.1 interactive interpreter\r
R:\e[9999C\e[6n\r>
R: \e[0K1 + 2 + 3\r
R:6\r
R:\e[9999C\e[6n\r>
R: \e[0K12345 + 23456 + 3\r> 2345 + 23456 + 34\e[0K\r> 345 + 23456 + 345\e[0K\r> 45 + 23456 + 3456\e[0K\r> 5 + 23456 + 34567\e[0K\r>  + 23456 + 34567 \e[0K\r> + 23456 + 34567 +\e[0K\r>  23456 + 34567 + \e[0K\r> 23456 + 34567 + 4\e[0K\r> 3456 + 34567 + 45\e[0K\r> 456 + 34567 + 456\e[0K\r> 56 + 34567 + 4567\e[0K\r> 6 + 34567 + 45678\e[0K
C:resize 10
R:\e[9999C\e[6n\r> 6 + 3456\e[1D\r> 6 + 3467\e[2D\r
R:84946\r
R:\e[9999C\e[6n\r>
R: \e[0K\r> + 45678\e[0K\r> + 2 + 3\e[0K\r
R:6\r
R:\e[9999C\e[6n\r>
R: \e[0K
EXIT:0
